worlds/soe/__init__.py

```diff
@@ -133,21 +132,28 @@ def _get_item_grouping() -> typing.Dict[str, typing.Set[str]]:
     return groups
 
 
+class SoEWebWorld(WebWorld):
+    theme = 'jungle'
+
+
 class SoEWorld(World):
     """
     Secret of Evermore is a SNES action RPG. You learn alchemy spells, fight bosses and gather rocket parts to visit a
     space station where the final boss must be defeated. 
     """
     game: str = "Secret of Evermore"
     options = soe_options
-    topology_present: bool = False
-    remote_items: bool = False
+    topology_present = False
+    remote_items = False
     data_version = 2
+    web = SoEWebWorld
```

Comment:
this is meant to be an instance of webworld, not the class. type checking should have flagged this. I wonder if we should explicitly confirm in the metaclass.

Reply:
just checked, i don't see a warning for it in my PyCharm -> Problems. Will push the `()` in a minute.